Automattic/jetpack
# Search package: dashboard use local gridicon

Fixes #22909 

#### Changes proposed in this Pull Request:
Replace the `gridicons` dependency with a local version in the Search package to reduce bundle size.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
- Spin up a new test site using this branch.
- Add a Search plan.
- Visit the Search dashboard at `/wp-admin/admin.php?page=jetpack-search`
- Confirm the Gridicons show up correctly on the page (see screenshot below).
- Toggle Instant Search.
- Confirm the notifications have Gridicons. There should be two, one for updating with an "info" icon and one for completion with a "checkmark" icon.

Screenshot showing Gridicons in the dashboard:

![Gridicons](https://user-images.githubusercontent.com/40267301/156999372-65b92566-baf3-417b-9ba7-d3c501261aaf.png)
